app/services/notifiers/rdv_base.rb

```diff
@@ -54,9 +54,13 @@ def generate_invitation_tokens
     # attributes or associations of the Rdv.
     @rdv.skip_webhooks = true
 
-    rdv_users_with_token_needed.each do |rdv_user|
+    tokens = rdv_users_with_token_needed.each do |rdv_user|
```

Comment:
Il me semble que `tokens` ne renvoie pas vraiment des tokens si tu itères avec `each`.
De toutes façons l'output de cette méthode n'est plus utilisé directement, on accède directement à `@rdv_users_tokens_by_user_id` depuis l'extérieur.